doc/python/renderers.md

```diff
@@ -223,12 +226,28 @@ fig = go.Figure(
 fig.show(renderer="png", width=800, height=300)
 ```
 
-### Displaying figures using Dash
-Dash is a Python framework for building web applications, and it provides built-in support for displaying Plotly figures. See the [Dash User Guide](https://dash.plot.ly/) for more information.
+### Displaying Figures Using Dash
+[Dash](https://dash.plot.ly) is a Python framework for building web applications, and it provides built-in support for displaying figures created with Plotly's graphing libraries. See the [Dash User Guide](https://dash.plot.ly/) for more information.
+
+It is important to note that Dash does not use the renderers framework discussed above, so you should not use the `show()` figure method or the `plotly.io.show` function inside Dash applications.
+
+Instead, pass your figure as the `figure` parameter to the `dcc.Graph` component. 
+
+```python
+import dash_core_components as dcc
```

Comment:
You need to add `dash_core_components` to `doc/requirements.txt` and `binder/requirements.txt` (the CI fails at the moment).